packages/babel-plugin-transform-private-methods/test/fixtures/accessors/basic/output.js

```diff
@@ -7,15 +7,15 @@ class Cl {
     this.publicField = "not secret string";
   }
   publicGetPrivateField() {
-    return babelHelpers.classPrivateGetter(this, _Cl_brand, _get_privateFieldValue);
+    return babelHelpers.classPrivateGetter(_Cl_brand, this, _get_privateFieldValue);
   }
   publicSetPrivateField(newValue) {
-    babelHelpers.classPrivateSetter(this, _Cl_brand, _set_privateFieldValue, newValue);
+    babelHelpers.classPrivateSetter(_Cl_brand, _set_privateFieldValue, this, newValue);
   }
 }
-function _get_privateFieldValue() {
-  return babelHelpers.classPrivateFieldGet2(this, _privateField);
+function _get_privateFieldValue(_this) {
+  return babelHelpers.classPrivateFieldGet2(_privateField, _this);
 }
-function _set_privateFieldValue(newValue) {
-  babelHelpers.classPrivateFieldSet2(this, _privateField, newValue);
+function _set_privateFieldValue(_this2, newValue) {
+  babelHelpers.classPrivateFieldSet2(_privateField, _this2, newValue);
 }
```

Comment:
I pushed another commit and now they can bind `_privateField` to save some size.

Reply:
I like this change -- passing `this` as a parameter also makes minification better because it can be renamed.

Reply:
I am doing a special case for `arguments`. :)